scss/civicrm/common/_editable-content.scss

```diff
@@ -0,0 +1,6 @@
+@import '../bootstrap/components/editable-content';
```

Comment:
what is this for?

Reply:
This is inheritance of editable content from bootstrap which shares same structure of markup.
![image](https://user-images.githubusercontent.com/26058635/33111455-35146394-cf74-11e7-993f-73decd78d00f.png)